cmd/kbn/node.go

```diff
@@ -109,31 +109,19 @@ func (n *Node) startRPC() error {
 		n.stopInProc()
 		return err
 	}
-	if n.config.IsFastHTTP() {
-		if err := n.startFastHTTP(n.httpEndpoint, apis, n.config.HTTPModules, n.config.HTTPCors, n.config.HTTPVirtualHosts); err != nil {
-			n.stopIPC()
-			n.stopInProc()
-			return err
-		}
-		if err := n.startFastWS(n.wsEndpoint, apis, n.config.WSModules, n.config.WSOrigins, n.config.WSExposeAll); err != nil {
-			n.stopHTTP()
-			n.stopIPC()
-			n.stopInProc()
-			return err
-		}
-	} else {
-		if err := n.startHTTP(n.httpEndpoint, apis, n.config.HTTPModules, n.config.HTTPCors, n.config.HTTPVirtualHosts); err != nil {
-			n.stopIPC()
-			n.stopInProc()
-			return err
-		}
-		if err := n.startWS(n.wsEndpoint, apis, n.config.WSModules, n.config.WSOrigins, n.config.WSExposeAll); err != nil {
-			n.stopHTTP()
-			n.stopIPC()
-			n.stopInProc()
-			return err
-		}
+
```

Comment:
I think we need to print WARN log if a user try to use `fasthttp` such as "fasthttp is deprecated, so start http server instead of fasthttp server"